DependencyInjection/Configuration.php

```diff
@@ -428,22 +428,31 @@ private function addModelSection(ArrayNodeDefinition $node)
     /**
      * @param ArrayNodeDefinition $node
      */
-    private function addBuzzSection(ArrayNodeDefinition $node)
+    private function addHttpClientSection(ArrayNodeDefinition $node)
     {
         $node
             ->children()
-                ->arrayNode('buzz')
+                ->arrayNode('http')
                     ->addDefaultsIfNotSet()
                     ->children()
-                        ->scalarNode('connector')->defaultValue('sonata.media.buzz.connector.curl')->end()
                         ->arrayNode('client')
                         ->addDefaultsIfNotSet()
                         ->children()
-                            ->booleanNode('ignore_errors')->defaultValue(true)->end()
-                            ->scalarNode('max_redirects')->defaultValue(5)->end()
+                            ->booleanNode('http_errors')->defaultValue(false)->end()
+                            ->arrayNode('allow_redirects')
```

Comment:
these things are what HttplugBundle is supposed to provide. ideally, this bundle would just have an option to set the httpclient service name. the default value would be `httplug.client.default` which happens when you configure nothing.

the doc can then link to http://docs.php-http.org/en/latest/integrations/symfony-full-configuration.html to explain how one can configure the client.